src/styles/components/bullet-hole.scss

```diff
@@ -0,0 +1,51 @@
+.bullet-hole {
+    position: absolute;
+    display: none;
+
+    &__1 {
+        transform: translateX(50px) translateY(20px);
```

Comment:
nice task for you -> create a mixin `transform` which will get 2 arguments and generate such a css rule ;) check docs for sass.